CogniteSdk.Types/Beta/DataModels/PropertyType.cs

```diff
@@ -168,6 +222,17 @@ public class DirectRelationPropertyType : BasePropertyType
         public ViewIdentifier Source { get; set; }
     }
 
+    /// <summary>
+    /// Property type for references to CDF resources.
+    /// </summary>
+    public class CDFExternalIdReference : BasePropertyType
+    {
+        /// <summary>
+        /// Whether this property is a list.
+        /// </summary>
+        public bool List { get; set; }
```

Comment:
We should probably name this `IsList` indicative of a boolean.
```suggestion
        public bool IsList { get; set; }
```

Reply:
This is a DTO.